### PR TITLE
Metrics modularization - mod-doc for Metrics Service

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
The upstream docs need to be refined before being brought downstream.

## What
Productization of Metrics work.

## Why
They cannot be brought downstream in their current state.

### CODE REVIEW

**Comment by jstaffor** [comment]
@finp  - this is ready for its initial review :+1:

File: modules/ROOT/pages/metrics.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+:installation-link: getting-started.html
+:docs-link: https://www.keycloak.org/documentation.html
+
+
+[id='a-collection-of-modules']
+
+= Using the {metrics-service} Mobile Service
+
+[[introduction]]
+The {metrics-service} service allows you to gather metrics on mobile apps, device versions, device security checks and back-end mobile service usage.
```
Comment (pmacko1) [inline modules/ROOT/pages/metrics.adoc]: text under toc without heading, do we generally want this @finp ?

Comment (finp) [reply modules/ROOT/pages/metrics.adoc]: @pmacko1  It makes sense ATM in order to get stuff downstream, worth revisiting after tech preview

File: modules/ROOT/pages/_partials/metrics/2_binding-an-app-to-a-service.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
++
+NOTE: It is possible to bind a {mobile-client} with a mobile service in the OpenShift console, however such bindings are not valid for the purposes of this procedure.
+
+. Click *Bind to App* for {metrics-service}.
+. Fill out the binding parameters required by the {metrics-service} Service.
+
++
+image::mobile-clients-services-idm-parameters.png[]
+NOTE: Use `Public` when binding a {mobile-client} to a {service-name}. When binding mobile services to each other, use `Bearer`.
```
Comment (finp) [inline modules/ROOT/pages/_partials/metrics/2_binding-an-app-to-a-service.adoc]: {service-name} isn't rendering, use {metrics-service} instead

File: modules/ROOT/pages/_partials/metrics/3_downloading-configuration-file.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* You have set up your mobile app development environment.
+
+.Procedure
+
+. Open your {mobile-client} in Mobile Developer Console.
+. Copy the `mobile-services.json` configuration to your clipboard.
+. Save the contents of the clipboard to a new file called `mobile-services.json`.
++
+NOTE: The `mobile-services.json` file is the link between your provisioned services on OpenShift and the mobile app you are developing. This file provides all required configuration to initialise the various SDKs and get them hooked up/connected to the back-end services.
+. Move mobile-services.json to the following location in your application project:
```
Comment (finp) [inline modules/ROOT/pages/_partials/metrics/3_downloading-configuration-file.adoc]: this is good, but needs to be followed up with removing corresponding lines (Move mobile-services.json ...) in tabs below

File: modules/ROOT/pages/_partials/metrics/7_monitoring-sync.adoc [hunk 4]
```diff
@@ -1,18 +1,29 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
 
+[id='monitoring-with-sync-{context}']
+= Monitoring the {sync-service} Service
```
Comment (finp) [inline modules/ROOT/pages/_partials/metrics/7_monitoring-sync.adoc]: @wei-lee will this still be valid with new data sync?

Comment (wei-lee) [reply modules/ROOT/pages/_partials/metrics/7_monitoring-sync.adoc]: yes, there will be a Grafana dashboard available for each sync app

**Comment by jstaffor** [comment]
@finp - the dynamite has been set. Light the fuse and send this PR to github HELL!

**Pull request merged**